src/chart/generateCategoricalChart.tsx

```diff
@@ -1802,19 +1802,6 @@ export const generateCategoricalChart = ({
       );
     };
 
-    renderPolarAxis = (element: any, displayName: string, index: number) => {
-      const axisType = get(element, 'type.axisType');
-      const axisMap = get(this.state, `${axisType}Map`);
```

Comment:
Nice and we now got rid of this string concatenation and reading stuff from state - that was super difficult to follow